modules/core/src/main/java/org/mycontroller/standalone/db/SensorUtils.java

```diff
@@ -103,6 +103,7 @@ public static String getValue(SensorVariable sensorVariable) {
             case V_STATUS:
                 data = McUtils.getStatusAsString(sensorVariable.getValue());
                 break;
+            case V_BRIGHTNESS:
```

Comment:
@Fraid No, you have to keep light_level as 0~100%, in backend we have to change the ratio, [like this](https://github.com/mycontroller-org/mycontroller/blob/development/modules/core/src/main/java/org/mycontroller/standalone/provider/rflink/RFLinkRawMessage.java#L160)

User always set value between 0~100, but actual value will be sent to node is our ratio.